kubernetes-maven-plugin/doc/src/main/asciidoc/inc/_introduction.adoc

```diff
@@ -56,11 +56,11 @@ The following table gives an overview of the different models
 | Default <<enrichers, Enrichers>> will create a default _Service_ and _Deployment_ (_DeploymentConfig_ for OpenShift) when no other resource objects are provided. Depending on the image they can detect which port to expose in the service. As with Generators, Enrichers support a limited set of configuration options.
 
 | *XML configuration*
-| jkube-m-p inherits the XML based configuration for building images from the https://jkubeio.github.io/docker-maven-plugin/#docker:build[docker-maven-plugin] and provides the same functionality. It supports an http://maven.apache.org/components/plugins/maven-assembly-plugin/assembly.html[assembly descriptor] for specifying the content of the Docker image.
+| k8s-m-p inherits the XML based configuration for building images from the https://jkubeio.github.io/docker-maven-plugin/#docker:build[docker-maven-plugin] and provides the same functionality. It supports an http://maven.apache.org/components/plugins/maven-assembly-plugin/assembly.html[assembly descriptor] for specifying the content of the Docker image.
```

Comment:
not using the same acronym defined in line 5 of this file (`k8-m-p`)
I would go for `k-m-p` or `k8s-m-p` (although the latter may lead to some confusion), but the acronym must be consistent across the complete documentation pages